envi/archs/arm/emu.py

```diff
@@ -35,19 +38,19 @@ def __init__(self):
         pass
 
     def stc(self, parms):
-        print >>sys.stderr,"CoProcEmu: stc(%s)"%repr(parms)
+        logger.info("CoProcEmu: stc(%s)"%repr(parms))
     def ldc(self, parms):
-        print >>sys.stderr,"CoProcEmu: ldc(%s)"%repr(parms)
+        logger.info("CoProcEmu: ldc(%s)"%repr(parms))
     def cdp(self, parms):
-        print >>sys.stderr,"CoProcEmu: cdp(%s)"%repr(parms)
+        logger.info("CoProcEmu: cdp(%s)"%repr(parms))
     def mcr(self, parms):
-        print >>sys.stderr,"CoProcEmu: mcr(%s)"%repr(parms)
+        logger.info("CoProcEmu: mcr(%s)"%repr(parms))
     def mcrr(self, parms):
-        print >>sys.stderr,"CoProcEmu: mcrr(%s)"%repr(parms)
+        logger.info("CoProcEmu: mcrr(%s)"%repr(parms))
     def mrc(self, parms):
-        print >>sys.stderr,"CoProcEmu: mrc(%s)"%repr(parms)
+        logger.info("CoProcEmu: mrc(%s)"%repr(parms))
     def mrrc(self, parms):
-        print >>sys.stderr,"CoProcEmu: mrrc(%s)"%repr(parms)
+        logger.info("CoProcEmu: mrrc(%s)"%repr(parms))
```

Comment:
once these are converted to the lower overhead logger syntax, this delta seems like it's ready to mainline.